### PR TITLE
Update disclaimer/LICENSE to reflect conda-build originally being part of conda

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

We were reminded (https://github.com/conda/conda-build/pull/4568#issuecomment-1353773401) that conda-build's code was originally part of conda and only later (~2014) was it moved into a separate repo.

This change uses standard SPDX syntax for disclaimers (I opted to use `©`, if there are any objections or concerns I'll switch it back to `Copyright (c)`), credits Continuum Analytics with the original copyright, and recognizes the rename to Anaconda in 2017.

Sought to otherwise standardize the disclaimer and LICENSE.txt to mirror that of conda (https://github.com/conda/conda/pull/12183).

Ref https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [ ] ~Add / update outdated documentation?~

### CODE REVIEW

File: .github/disclaimer.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-Copyright (C) 2014 Anaconda, Inc
+SPDX-FileCopyrightText: © 2012 Continuum Analytics, Inc. <http://continuum.io>
+SPDX-FileCopyrightText: © 2017 Anaconda, Inc. <https://www.anaconda.com>
```
Comment (mbargull) [inline .github/disclaimer.txt:2]: Let's not overcomplicate things and go with
```suggestion
SPDX-FileCopyrightText: © 2012 Anaconda, Inc. <https://www.anaconda.com>
```
The copyright remained with the same entity ([CIK: 0001538860](https://www.sec.gov/edgar/browse/?CIK=0001538860)), after all.
Let's just us the current name.

**Review by mbargull — CHANGES_REQUESTED** [review]

**Comment by jezdez** [comment]
Hmm, I wonder if https://hynek.me/til/copyright-years/ might be worth another look, a lot of corporate project have removed the years altogther!

**Comment by mbargull** [comment]
> Hmm, I wonder if https://hynek.me/til/copyright-years/ might be worth another look, a lot of corporate project have removed the years altogether!

Ah, nice find! It's good to see that this is has some notable examples. That'd be my favored solution because it's more faithful, https://github.com/conda/conda-build/pull/4568#issuecomment-1353773401 :
> However, a lot of the code/source files are from 2016+, i.e., `(c) 2012`/`(c) 2014` is not representative/true for that.

Going with `SPDX-FileCopyrightText: © Anaconda, Inc. <https://www.anaconda.com>` in the file headers and either leaving `LICENSE.txt` untouched or just substituting Continuum -> Anaconda sounds reasonable.
(Though, I'd leave `2012` in `LICENSE.txt` -- doesn't really hurt at that single place, plus it is a faithful statement in there.)

File: .github/disclaimer.txt (hunk 1 above)
Comment (mbargull) [reply .github/disclaimer.txt:2]:
```suggestion
SPDX-FileCopyrightText: © Anaconda, Inc. <https://www.anaconda.com>
```

**Comment by jezdez** [comment]
No, this is blocked by the legal work being done in providing a trademark license to NumFOCUS and can now picked up again.

**Pull request closed without merging**